scripts/datenlord-monitor-deploy.sh

```diff
@@ -4,29 +4,33 @@ set -o errexit
 set -o nounset
 set -o xtrace
 
-if [ "$1" = "helm" ]
+if [ ${1-default} = "helm" ]
 then
-    kubectl create namespace datenlord-monitoring 
+    kubectl create namespace datenlord-monitoring
+    kubectl create namespace datenlord-logging 
     helm repo add stable https://charts.helm.sh/stable
     helm install prometheus stable/prometheus-operator --namespace datenlord-monitoring 
     helm repo add elastic https://helm.elastic.co
-    helm install filebeat elastic/filebeat
-    helm install kibana elastic/kibana
-    helm install elasticsearch elastic/elasticsearch -f elasticsearch-values.yaml
+    helm install filebeat elastic/filebeat --namespace datenlord-logging 
+    helm install kibana elastic/kibana --namespace datenlord-logging 
+    helm install elasticsearch elastic/elasticsearch -f elasticsearch-values.yaml --namespace datenlord-logging 
     kubectl wait --for=condition=Ready pod -l app.kubernetes.io/name=grafana -n datenlord-monitoring --timeout=60s
-    kubectl wait --for=condition=Ready pod -l app=elasticsearch-master --timeout=120s
-    kubectl wait --for=condition=Ready pod -l app=kibana --timeout=120s
-    kubectl wait --for=condition=Ready pod -l app=prometheus --timeout=120s
+    kubectl wait --for=condition=Ready pod -l app=elasticsearch-master -n datenlord-logging --timeout=120s
+    kubectl wait --for=condition=Ready pod -l app=kibana -n datenlord-logging --timeout=120s
+    kubectl wait --for=condition=Ready pod -l app=prometheus -n datenlord-logging --timeout=120s
     POD_NAME=`kubectl get pods -l app.kubernetes.io/name=grafana | grep grafana | awk '{print $1}'`
     kubectl port-forward $POD_NAME $GRAFANA_PORT -n datenlord-monitoring &
+elif [ ${1-default} = "undeploy" ]
+then
+    kubectl delete all --all -n datenlord-monitoring
+    kubectl delete all --all -n datenlord-logging
```

Comment:
Will this clean everything by destroying namespace?

Reply:
Yes